FoxHound.Client/package.json

```diff
@@ -20,12 +21,14 @@
   "dependencies": {
     "@material-ui/core": "^4.9.8",
     "@material-ui/icons": "^4.9.1",
+    "@material/top-app-bar": "^5.1.0",
```

Comment:
Why was this @material/top-app-bar package added? Was the @material-ui/core not enough to accomplish adding the navigation? I'm not sure i see this package referenced anywhere in the code either.

Reply:
That was my initial attempt in trying to get the Nav Bar implemented.  I ended up abandoning it.  I must not have taken it out of the package.json file as a dependency.